contracts/service-provider-directory/src/state/admin.rs

```diff
@@ -0,0 +1,15 @@
+use cosmwasm_std::{Addr, Deps, DepsMut};
+use cw_controllers::Admin;
+
+use crate::error::Result;
+
+const ADMIN_KEY: &str = "admin";
```

Comment:
for consistency sake with other contracts (and to make it easier to spot accidental clashes) - can you move storage keys to `constants.rs`?